src/test/java/org/opensearch/security/dlic/rest/api/AccountApiTest.java

```diff
@@ -222,9 +225,9 @@ public void testPutAccountRetainsAccountInformation() throws Exception {
         response = rh.executeGetRequest(internalUserEndpoint);
         assertEquals(HttpStatus.SC_OK, response.getStatusCode());
         Settings responseBody = Settings.builder()
-                 .loadFromSource(response.getBody(), XContentType.JSON)
-                 .build()
-                 .getAsSettings(testUsername);
+		.loadFromSource(response.getBody(), XContentType.JSON)
```

Comment:
Revert back to the original formatting prior to PR #1172.

Reply:
Done